src/app/view/components/dot-contentlet-editor/components/dot-add-contentlet/dot-add-contentlet.component.spec.ts

```diff
@@ -83,6 +83,7 @@ describe('DotAddContentletComponent', () => {
             spyOn(component, 'onClose').and.callThrough();
```

Comment:
All this PR is about Bryan's comment here: https://github.com/dotCMS/core/issues/14334#issuecomment-390763001, screencast: https://screencast.com/t/GKTt6iYw